autodial-worker/src/worker.ts

```
@@ -0,0 +1,128 @@
+import { Env } from './types';
+import { getGoal, getGoals, updateGoal, dial, getSettings, now, SID } from './lib';
+
+export interface Env {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Env'. Consider to delete it or rename it.

'Env' is defined here:


(lint/suspicious/noRedeclare)